pkg/types/baremetal/validation/libvirt.go

```diff
@@ -45,20 +45,26 @@ func interfaceValidator(libvirtURI string) (func(string) error, error) {
 		return nil, errors.Wrap(err, "could not connect to libvirt")
 	}
 
-	interfaces, err := conn.ListAllInterfaces(libvirt.CONNECT_LIST_INTERFACES_ACTIVE)
+	networkNames, err := conn.ListNetworks()
+	if err != nil {
+		fmt.Print("could not list libvirt networks")
```

Comment:
```suggestion
		return nil, errors.Wrap(err, "could not list libvirt networks")
```

Reply:
yes the change is needed in order to use libvirt networks, beyond the interface bridge ones